doc/source/python_reference.rst

```diff
@@ -139,7 +139,7 @@ Functions:
    read_bem_solution
    read_bem_surfaces
    read_cov
-   read_dip
+   read_dipoles
```

Comment:
> We should create a Dipole class with a plot method
This is interesting for all sparse inverse methods. For the plot function, we should think of a way to plot not only the location but also the orientation per per dipole and time point similar to the way of selecting a time point in our visualization of dense solvers